components/AddForm/AddForm.tsx

```diff
@@ -13,33 +13,27 @@ const AddForm = ({ person }: { person: Resident }): React.ReactElement => {
   const [url, setUrl] = useState<string>();
   const ageContext = person && person.contextFlag;
   const forms = ageContext === 'C' ? CHILD_FORMS : ADULT_FORMS;
-  const internalForms =
-    ageContext === 'A'
-      ? [
-          {
-            text: 'Case Note Recording',
-            value: `/people/${person.id}/records/case-notes-recording`,
-          },
-          // {
-          //   text: 'Conversation 3',
-          //   value: `/people/${person.id}/records/conversation-3`,
-          // },
```

Comment:
Do we know why this was commented out? Are we still working on `Conversation 3`? 
Wondering if we should keep the commented out code in case someone goes looking for it? 😁

Reply:
I'm happy to get rid of commented code, it's always available in version control should someone need to go back to it!